TeamCode/src/main/java/org/firstinspires/ftc/teamcode/bots/FrenzyBaseBot.java

```diff
@@ -83,7 +83,7 @@
     public static final double ROBOT_CENTER_Y = 8.25;
 
 
-    private BotCalibConfig botConfig;
+    protected BotCalibConfig botConfig;
```

Comment:
there is a public accessor method for botConfig, but it is OK to make it protected